### PR TITLE
[RFR] better translation...

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

File: src/app/js/i18n/translations/fr.js [hunk 1]
```diff
@@ -103,8 +103,8 @@ export default {
     restore: 'Restaurer',
     removed_at: 'Removed at',
     removed_reason: 'La raison',
-    add_field_to_resource: 'Ajoutez une champs à cette ressource',
-    about_you: 'Rédigez votre "À propos":',
+    add_field_to_resource: 'Ajouter un champs à cette ressource',
```
Comment (parmentf) [inline src/app/js/i18n/translations/fr.js]: Je crois qu'au singulier, `champ` ne prend pas de `s`.
Voir http://chouxdesiam.canalblog.com/archives/2007/07/12/5604191.html et https://fr.wikipedia.org/wiki/Champ

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
What's left for RFR?

**Comment by parmentf** [comment]
There remains untranslated keys.

**Comment by fzaninotto** [comment]
Needs rebase. Also, please close PRs within a day or two, or they require more and more work over time. If you're missing translations, add them in another PR.

**Pull request merged**